yarn.lock

```diff
@@ -4223,6 +4231,27 @@ can-use-dom@^0.1.0:
   integrity sha1-IsxKNKCrxDlQ9CxkEQJKP2NmtFo=
 
 caniuse-db@^1.0.30000977:
+<<<<<<< HEAD
```

Comment:
@vishweshjha Fix this conflicts you have committed. Just delete the file and re run yarn